Scripts/setup_private_specs.sh

```diff
@@ -90,7 +94,9 @@ while read -r podspec_file ; do
   podspec_file_names+=("$podspec_file_name")
   pod_name=$( basename "$podspec_file_name" .podspec )
 
-  dst_dir="${COCOAPODS_REPO_DIR}/${pod_name}/${new_version}"
+  resolved_version=$( resolve_version_for_pod "${pod_name}" )
```

Comment:
when i first read this, i thought we were passing in "${pod_name}" as an argument to `resolve_version_for_pod`, but instead it looks like we are just appending the two together -- i think it would be more readable if we broke this into multiple lines -- one line for calling the funciton, and another for appending the "${pod_name}"

Reply:
We are passing `pod_name` as an argument to `resolve_version_for_pod`. It's there as a hook for pods that need special handling, like `AWSCognitoIdentityProviderASF` used to. It's actually safe to remove it now, but it's a useful extension point.